sip-ri/src/test/java/gov/nist/javax/sip/parser/extensions/TargetDialogParserTest.java

```diff
@@ -7,20 +7,12 @@ public class TargetDialogParserTest extends ParserTestCase {
     @Override
     public void testParser() {
         String[] targetDialogHeaders = {
-                "Target-dialog: f0b40bcc-3485-49e7-ad1a-f1dfad2e39c9@10.5.0.53\r\n",
-                "Target-dialog: f81d4fae-7dec-11d0-a765-00a0c91e6bf6@foo.bar.com\r\n",
-                "Target-dialog: 1@10.0.0.1\r\n",
-                "Target-dialog: kl24ahsd546folnyt2vbak9sad98u23naodiunzds09a3bqw0sdfbsk34poouymnae0043nsed09mfkvc74bd0cuwnms05dknw87hjpobd76f\r\n",
-                "Target-dialog: 281794\r\n",
-        		//local tag
-                "Target-dialog: f0b40bcc-3485-49e7-ad1a-f1dfad2e39c9@10.5.0.53\r\n",
-                //remote tag 
-                "Target-dialog: f81d4fae-7dec-11d0-a765-00a0c91e6bf6@foo.bar.com\r\n",
-                // local tag 
-                "Target-dialog: kl24ahsd546folnyt2vbak9sad98u23naodiunzds09a3bqw0sdfbsk34poouymnae0043nsed09mfkvc74bd0cuwnms05dknw87hjpobd76f@example.com\r\n" 
+        		 "Target-dialog: f0b40bcc-3485-49e7-ad1a-f1dfad2e39c9@10.5.0.53\n", 
+	             "Target-dialog: f81d4fae-7dec-11d0-a765-00a0c91e6bf6@foo.bar.com\n",
+	             "Target-dialog: kl24ahsd546folnyt2vbak9sad98u23naodiunzds09a3bqw0sdfbsk34poouymnae0043nsed09mfkvc74bd0cuwnms05dknw87hjpobd76f\n",
+	             "Target-dialog: 12345th5z8z;local-tag=tozght6-45;remote-tag=fromzght789-337-2\n"
```

Comment:
You should add a check in the test to do toString and make sure it prints the same string or values. Given the implementation right now it would print from-tag and to-tag, not local-tag nor remote-tag

Reply:
fixed it